Project.toml

```diff
@@ -3,7 +3,7 @@ uuid = "c7f686f2-ff18-58e9-bc7b-31028e88f75d"
 keywords = ["markov chain monte carlo", "probablistic programming"]
 license = "MIT"
 desc = "Chain types and utility functions for MCMC simulations."
-version = "4.13.2"
+version = "4.17.0"
```

Comment:
Why is this version bumped so much and why isn't it registered? @cpfiffer

Reply:
Woops an oversight on my part. The overbump was because there's other PRs that needed to be merged and it wasn't clear which was first. I'll fix and tag this.